frontend/components/tasks/affectiveAnnotation/inputs/TextfieldModal.vue

```diff
@@ -95,28 +95,21 @@ export default {
       emptyTextFlag: '-',
       enableTextfield: true,
       showDialog: false,
-      dialogErrorMessage: ''
-    }
-  },
-
-  computed: {
-    text: {
-      get() {
-        return this.value === this.emptyTextFlag ? '' : this.value
-      },
-      set(val) {
-        this.$emit('input', val)
-      }
+      dialogErrorMessage: '',
+      text: (this.value === this.emptyTextFlag) ? '' : this.value
```

Comment:
The `data` field is only executed once before mounting the component. I haven't tested this, but from my understanding, this should return error on every tries after the 1st try, did you check? I think it's better to implement the `watch` block to watch the `value` and change the `text` value based on the `value`. There is an example somewhere in the code base

Reply:
It actually appeared to work fine on multiple tests, but I understand your point. Basically `text` was only filled by `value` at the beginning before mounting, and after that they work independently from each other. Even though they basically have the same value all the time, it is quite prone to bugs, so I agree that using `watch` on `value` to change `text` is better. I have refactored it, thank you for the feedback :)